core/utils/fs.js

```diff
@@ -41,19 +42,15 @@ async function sendToTrash(fullpath /*: string */) {
   try {
     await shell.trashItem(fullpath)
   } catch (err) {
-    if (
-      err.message === 'Failed to move item to trash' || // Error message on Linux
-      err.message === 'Failed to parse path' || // Error message on Windows
-      /doesn’t exist/.test(err.message) // Error message on macOS
-    ) {
```

Comment:
I don't understand. The commit message says that the detection mechanism is changed only for macOS, but it looks like two lines for Linux and Windows are removed too.

Reply:
Right. This fix is mainly for macOS as the detection works fine on Linux and Windows but since the behavior will be different on all OS I should update the message.

Reply:
Also, I forgot to update the PR's message when I updated the commit's message. It must have been really confusing indeed.